test/simulator/items/whiteherb.js

```diff
@@ -0,0 +1,150 @@
+'use strict';
+
+const assert = require('./../../assert');
+const common = require('./../../common');
+
+let battle;
+
+describe('White Herb 5678765567', function () {
```

Comment:
Debugging artifact, I presume?

```suggestion
describe('White Herb', function () {
```

Reply:
yup :D will remove